pkgs/top-level/all-packages.nix

```diff
@@ -12762,7 +12762,6 @@ in
   cmakeCurses = cmake.override { useNcurses = true; };
 
   cmakeWithGui = cmakeCurses.override { withQt5 = true; };
-  cmakeWithQt4Gui = cmakeCurses.override { useQt4 = true; };
```

Comment:
Missing an entry to aliases.nix.

Reply:
bump. Please add a `cmakeWithQt4Gui = throw "..."` entry in aliases.nix